src/components/ListItem.jsx

```diff
@@ -29,9 +29,10 @@ export function ListItem({ item, listToken }) {
 			console.log('error', error);
 		}
 	};
+	// getDaysBetweenDates of lastPurchasedDate and now
 
 	useEffect(() => {
-		if (timeElapsed > one_day_in_ms) {
+		if (timeElapsed > one_day_in_ms && item.isChecked === true) {
```

Comment:
Nice job catching the bug from last week and implementing this extra conditional!